apps/fabric-website/src/styles/_mixins.scss

```diff
@@ -38,9 +40,9 @@
   position: relative;
 
   &::after {
-    bottom: 0;
+    bottom: -1px;
```

Comment:
This is probably fine, but 1px adjustments like this sometimes worry me a bit since there can be browser rendering differences that will change how this shows up.

Reply:
@Jahnp This bug is one that appeared in the emulator yesterday in Chrome. I can check other browsers, but I can also remove it, as the bug appears in limited scales and is not super noticeable. Let me know if you want to see the difference in action on my local machine.

Reply:
It should be fine to keep--just wanted to call it out as a potential browser discrepancy.